indico/modules/events/editing/schemas.py

```diff
@@ -162,3 +162,24 @@ def _check_for_unique_tag_code(self, code):
             query = query.filter(EditingTag.id != tag.id)
         if query.has_rows():
             raise ValidationError(_('Tag code must be unique'))
+
+
+class EditableFileTypeArgs(mm.Schema):
+    class Meta:
+        rh_context = ('event', 'file_type')
+
+    name = fields.String(required=True, validate=not_empty)
+    extension = fields.List(required=True)
```

Comment:
specify the type of the list members: `fields.List(fields.String(), required=True)`